autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyResolver.java

```diff
@@ -22,6 +33,36 @@ public MavenDependencyResolver() {
         Model model = mavenModelAnalyser.getModel(workspace);
         Set<Dependency> dependencies = getDependencies(model);
         dependencies.addAll(getPlugins(model));
+        var modules = model.getModules();
+        if (!modules.isEmpty()) {
+            try {
```

Comment:
you can remove this try catch into resolveModules after that this bit of code can get a lot more readable